addons/web/static/tests/views/kanban/kanban_view_tests.js

```diff
@@ -10634,7 +10632,7 @@ QUnit.module("Views", (hooks) => {
         // -> for the record matching the ID, the base64 should be returned
         // -> for all the other records, the image should be displayed by url
         const imageOnRecord = target.querySelectorAll(
-            'img[data-src*="/web/image"][data-src*="&id=1"]'
+            'img[data-src*="/web/image/partner/1/image"]'
```

Comment:
prefer full assert rather than includes (I want to see the unique key in all tests)